docs/configuration.md

```diff
@@ -714,6 +714,13 @@ Apart from these, the following properties are also available, and may be useful
 <table class="table">
 <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
 <tr>
+  <td><code>spark.eventLog.blockUpdatesEnabled</code></td>
```

Comment:
We generally use ".enabled" for boolean options. Another possibility is calling this `logBlockUpdates`, which might be more self-describing.

Reply:
How about `spark.eventLog.logBlockUpdates.enabled`?

Reply:
Sure.